editor/editor_help_search.cpp

```diff
@@ -165,11 +186,11 @@ void EditorHelpSearch::popup_dialog(const String &p_term) {
 
 	if (p_term.is_empty()) {
 		search_box->clear();
+
+		old_search = true;
```

Comment:
Seems a bit weird that an empty term and cleared search box equals `old_search = true;`

Reply:
This is to force a search if the popup closes and reopens with a short string, I'm currently seeing how best to resolve that